codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/middleware/MiddlewareBuilder.java

```diff
@@ -186,6 +186,7 @@ private Collection<? extends ClientConfig> getDefaultClientConfig() {
         ClientConfig logger = ClientConfig.builder()
                 .name("logger")
                 .type("Logger")
+                .rbsType("untyped") // TODO: find out why Logger does not resolve
```

Comment:
Rbs seems to be missing some core classes..... my guess is its missing this one :-(

Reply:
Created issues in rbs